chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -605,6 +605,15 @@ def miimon=(millisecs)
       self
     end
 
+    def xmit_hash_policy
+      sysfs("bonding/xmit_hash_policy")
```

Comment:
At least in my testing, it's always present, and you can in fact always set it -- I suspect the kernel ignores the setting if the mode is not 2, 4 or 5.